app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -209,8 +209,9 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService)(implic
       case Some(document) =>
         val applicationId = document.getAs[String]("applicationId").get
         val applicationStatus = document.getAs[String]("applicationStatus").get
+        val fastPassReceived = document.getAs[FastPassDetails]("fastpass-details").flatMap(_.fastPassReceived)
         findProgress(applicationId).map { (p: ProgressResponse) =>
-          ApplicationResponse(applicationId, applicationStatus, userId, p)
+          ApplicationResponse(applicationId, applicationStatus, userId, p, fastPassReceived)
```

Comment:
I am thinking maybe fastPassReceived is a kind of status or progress? I don't like the idea of storing that outside applicationStatus or progressStatus. And I know we will have like that, maybe we have to think how to deal with it.